### DISCUSSION TITLE
Arbeit beendet bei Silo/Bunker lader

### DISCUSSION THREAD

**Discussion by cabmac** [Courseplay/Courseplay_FS22#3113] [Q&A]
Hi...seit ein paar Tagen laeuft mein Radler nicht mehr im Bunkerlader-Modus.. Er bleit immer an der Zielposition stehen mit der Meldung "Arbeit beendet" ohne halt weiter in das Silo zu fahren ,zu laden und im Bunker abzuladen, wie er es sonst immer tat....ich dachte ,es haette sich vielleicht etwas verstellt, aber nein...er faehrt einfacj nicht mehr in das Silo...einer eine Idee?

**Comment by Tensuko** [comment]
Mehr Infos und vlt ein Bild vom Bunker mit dem Radlader, eventuell sieht man da was.

**Reply by cabmac** [reply]
hat sich erledigt..das Silo hatte zwar noch so ca. 100000l Mais,aber es wurde Fuellstand o angezeigt,dass hat wohl dann der Radlader auch angenommen..:). warum auch immer...nichts fuer ungut
